AEPTestUtils.podspec

```diff
@@ -18,4 +18,5 @@ Pod::Spec.new do |s|
   s.dependency 'AEPCore', '>= 4.0.0'
 
   s.source_files = 'Sources/**/*.swift'
+  s.frameworks   = 'XCTest'
 end
```

Comment:
we should target both ios and tvos.deployment_target = '11.0' similar to the SDK

Reply:
Can you also check if this also matches the settings in xcode project for iOS_DEPLOYMENT_TARGET and tvOS_DEPLOYMENT_TARGET and Package.swift?